src/lib/discord/commands/_repositories.ts

```diff
@@ -1,23 +1,94 @@
-/**
- * @TODO replace with GitHub API call
- */
+import { getRepos } from '../../github/queries'
+
+const nameToAlias = new Map<string, string>([
+  ['amplify-cli', 'cli'],
+  ['amplify-ios', 'ios'],
+  ['discord-bot', 'discord-bot'],
+  ['amplify-ui', 'ui'],
+  ['amplify-js', 'js'],
+  ['docs', 'docs'],
+  ['amplify-android', 'android'],
+  ['amplify-flutter', 'flutter'],
+  ['amplify-category-api', 'category-api'],
+  ['amplify-cli-export-construct', 'cli-export-construct'],
+  ['amplify-codegen', 'codegen'],
+  ['amplify-adminui', 'adminui'],
+  ['amplify-hosting', 'hosting'],
+  ['Test-Repository', 'TEST'],
+])
+
+const alias = (name: string) =>
+  nameToAlias.get(name) ? nameToAlias.get(name) : name
```

Comment:
```suggestion
const alias = (name: string) => nameToAlias.get(name) ?? name
```

since this value will be `undefined` if it doesn't exist, we can use `||` or [`??`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) to specify a fallback. I don't think we have a good case now for `null` values in the alias map, so either would be fine, however perhaps later we'll want to define a repo with an explicit `null` alias to force the fallback